_docs_use-cases/use-case-consumption-notify-peer-about-repositoryattribute-succession.md

```diff
@@ -62,9 +62,10 @@ Moreover, it is also possible to notify a peer about the succession of an Identi
 
 ## On Failure
 
-- The response cannot be created, if the `peer` is unknown.
-- The response cannot be created, if the `attributeId` belongs to a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute).
-- The response cannot be created, if the `attributeId` belongs to an IdentityAttribute with a `shareInfo`.
-- The response cannot be created, if you already notified the peer about the succession of the Attribute.
-- The response cannot be created, if you haven't shared a version of the Attribute with the peer before. To initially share an IdentityAttribute, use the [ShareIdentityAttribute use-case]({% link _docs_use-cases/use-case-consumption-share-a-repositoryattribute.md %}).
-- The response cannot be created, if the parameters are malformed.
+- The response cannot be created if the `peer` is unknown.
+- The response cannot be created if the `attributeId` belongs to a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute).
+- The response cannot be created if the `attributeId` belongs to an IdentityAttribute with a `shareInfo`.
+- The response cannot be created if you already notified the peer about the succession of the Attribute.
```

Comment:
Be more precise here: " The response cannot be created if you already notified the peer about the succession of the Attribute **or a newer version of it**."

Reply:
In my opinion, "the succession of the Attribute" also includes later versions of the Attribute. Note that we use the term "Attribute" to refer to all versions of an Attribute. For me your propose version is actually more confusing.

Reply:
I can understand that this formulation can be a bit confusing. However, perhaps we should then (does not have to be done in this PR) add the bullet point "The response cannot be created if you already shared this or a newer version of the RepositoryAttribute with the peer." here as well, as the succession of a RepositoryAttribute and the notification of the peers about the succession do not have to be related in terms of time for Connector-Connector communication (in contrast to App-Connector communication).